.github/workflows/prebuild.yml

```diff
@@ -75,23 +75,73 @@ jobs:
           mv target/wasm32-wasip2/debug/datafusion_udf_wasm_python.wasm   out/datafusion_udf_wasm_python.debug.wasm
           mv target/wasm32-wasip2/release/datafusion_udf_wasm_python.wasm out/datafusion_udf_wasm_python.release.wasm
 
+      - name: compile WASM to machine code
+        run: |
+          targets=(
+            "aarch64-apple-darwin"
+            "aarch64-pc-windows-msvc"
+            "aarch64-unknown-linux-gnu"
+            "x86_64-pc-windows-msvc"
+            "x86_64-unknown-linux-gnu"
+          )
+
+          for wasm_file in out/*.wasm; do
+            wasm_file="$(basename "$wasm_file" .wasm)"
+            for target in "${targets[@]}"; do
+              out_file="${wasm_file}.${target}.so"
```

Comment:
Should these always be `so` files? I'd expect `dynlib` on Mac and `dll` on windows, but I might be expecting the wrong thing.